templates/katello.yaml.erb

```diff
@@ -23,9 +23,11 @@
     :ssl_key_file: <%= @candlepin_events_ssl_key %>
     :ssl_ca_file: <%= @candlepin_ca_cert %>
 
+<% if @pulp2_support %>
   :pulp:
     :url: <%= @pulp_url %>
     :ca_cert_file: <%= @pulp_ca_cert %>
+<% end %>
 
   :use_pulp_2_for_content_type:
```

Comment:
We probably should drop this section too.

Reply:
And ensure   :katello_applicability: <%= !@use_pulp_2_for_yum %> is true

Reply:
@jlsherrill what happens to https://github.com/theforeman/puppet-katello/pull/363/files#diff-c48970cf4dfa5d2a2124a59cac2bf934R40 in a pure Pulp 3 scenario?